themes/root/templates/HelpTranslations/en/geosearch.phtml

```diff
@@ -11,11 +11,11 @@
     <p>Search results are displayed on the map as clusters (circular icons with numbers). The numbers represent
         the number of records in each cluster. A list of results is displayed below the map view. </p>
     <p>You can interact with the map by zooming by either using the zoom buttons (+/-) or by using the mouse to
-        double-click (zoom in) or shift + double-click (zoom out).</p>
-    <p>Clusters with less than 5 records will display pop-up information about the records at that location.
-        When you hover over a cluster with less than 5 records, the mouse pointer will change to a hand with a finger
-        pointing at the cluster. Click on the cluster, and a pop-up window will appear showing a list of titles for the
-        records at that location. Each title is hyperlinked to the associated record.
-        Click on the title to view the full record.</p>
+        double-click (zoom in) or shift + double-click (zoom out). You can also click on a cluster to zoom to 
```

Comment:
@xmorave2, the geographic interface is changing slightly due to the switch to Leaflet instead of OpenLayers. Would you have a moment to adjust the Czech translation of this file to match?